.github/workflows/build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         with:
           egress-policy: audit
       - id: setup-env
-        uses: cisagov/setup-env-github-action@develop
+        uses: cisagov/setup-env-github-action@improvement/add_goimports_version
```

Comment:
Make sure you don't forget to change this back!

Reply:
I have rebased to drop the commit now that https://github.com/cisagov/setup-env-github-action/pull/72 has been merged.